src/migrations/20201130184519-rename-user-title-column.js

```diff
@@ -0,0 +1,50 @@
+const titles = [
+  'Program Specialist',
+  'Early Childhood Specialist',
+  'Grantee Specialist',
+  'Family Engagement Specialist',
+  'Health Specialist',
+  'Systems Specialist',
+];
+
+const roles = [
+  'Regional Program Manager',
+  'COR',
+  'Supervisory Program Specialist',
+  'Program Specialist',
+  'Grants Specialist',
+  'Central Office',
+  'TTAC',
+  'Admin. Assistant',
+  'Early Childhood Manager',
+  'Early Childhood Specialist',
+  'Family Engagement Specialist',
+  'Grantee Specialist Manager',
+  'Grantee Specialist',
+  'Health Specialist',
+  'System Specialist',
+];
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    queryInterface.removeColumn('Users', 'title');
+    queryInterface.addColumn(
+      'Users',
+      'role',
+      {
+        type: Sequelize.DataTypes.ENUM(...roles),
+      },
+    );
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    queryInterface.addColumn(
+      'Users',
+      'title',
+      {
+        type: Sequelize.DataTypes.ENUM(...titles),
+      },
+    );
+    queryInterface.removeColumn('Users', 'role');
+  },
+};
```

Comment:
Renaming to "role" makes sense. This might also change going forward, since we are introducing the "Role" table. If sticking with enum for now, we will also need to drop the enum type. Otherwise, there will be an error trying to undo the migration.

Something similar to this can be used (although untested in migrations):
```
const query = 'DELETE FROM pg_type WHERE typname = \'enum_Users_title\' ';
    return queryInterface.sequelize.query(query);
```
The same goes for `enum_Users_role`